### PR TITLE
Fix plupload file field listings

### LINKED ISSUE #3731: Fix plupload file field listings
It seems that the handlebars template has a hidden class which may not get removed for existing items if it uses `!important`. For new uploads the slide down does not override the hidden class as well if it uses `!important`.

Needs a compatibility fix to use `hide()`, remove `hidden`, and then slideDown.

### PR DESCRIPTION
**Pull request by pcfreak30**
Fixes #3731

### CODE REVIEW

**Comment by sc0ttkclark** [comment]
Can you merge latest 2.x and resolve conflicts in plupload.php? Thanks!

**Comment by pcfreak30** [comment]
Conflict resolved

File: ui/fields/media.php [hunk 1]
```diff
@@ -293,7 +295,7 @@
                         if ( !$list_<?php echo esc_js( pods_js_name( $attributes[ 'id' ] ) ); ?>.is( ':visible' ) )
                             $list_<?php echo esc_js( pods_js_name( $attributes[ 'id' ] ) ); ?>.show().removeClass( 'hidden' );
 
-                        $list_<?php echo esc_js( pods_js_name( $attributes[ 'id' ] ) ); ?>.find( 'li.pods-file:first' ).slideDown( 'fast' );
+                        $list_<?php echo esc_js( pods_js_name( $attributes[ 'id' ] ) ); ?>.find( 'li.pods-file:first' ).hide().removeClass('hidden')..slideDown( 'fast' );
```
Comment (pglewis) [inline ui/fields/media.php]: Breaks file upload as there are two periods before `slideDown`

**Comment by pglewis** [comment]
64b18c breaks file upload with a JavaScript error, note in the commit.

File: ui/fields/media.php (hunk 1 above)
Comment (sc0ttkclark) [inline ui/fields/media.php]: Fixed via hulk smash https://github.com/pods-framework/pods/commit/30dffaaf8cb8d530c6a30434af783d6c826760a6

**Pull request merged**